.github/workflows/workflow.yml

```diff
@@ -8,51 +8,7 @@ on:
   pull_request:
 
 jobs:
-  common_checks:
```

Comment:
why are you reverting this change?

Reply:
I'm curious about this change, cause it supposed to be in develop, and this file was synced with develop branch, probably my develop branch was not synced at the moment